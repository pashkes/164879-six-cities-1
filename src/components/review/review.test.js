import React from "react";
import {shallow, configure} from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import Review from "./review.jsx";

configure({adapter: new Adapter()});


const reviewMock = {
  id: `0`,
  comment: ``,
  date: ``,
  machineDate: ``,
  rating: 0,
  name: ``,
  avatar: ``,
  isPro: false,
};

it(`review renders correctly`, () => {
  const review = shallow(<Review {...reviewMock} />);
  expect(review).toMatchSnapshot();
});
