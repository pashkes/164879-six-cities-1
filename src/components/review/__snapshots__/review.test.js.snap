// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`review renders correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt=""
        className="reviews__avatar user__avatar"
        height="54"
        src=""
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    />
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "0%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    />
    <time
      className="reviews__time"
      dateTime=""
    />
  </div>
</li>
`;
